### DISCUSSION TITLE
What  property is spell  by player in player Model?

### DISCUSSION THREAD

**Discussion by QQ2287991080** [fraxiinus/ReplayBook#186] [General]
![image](https://user-images.githubusercontent.com/48544629/163763025-f09e9691-6cf7-4b33-bb57-0827e98a5deb.png)

thanks

**Comment by fraxiinus** [comment]
SPELLX_CAST is the amount of times a player used X spell. Where 1 = Q, 2 = W, and so on.

SUMMON_SPELL1_CAST is the amount of times a player used their first summoner spell; like flash or ignite. Unfortunately it is not possible to tell which summoner spells a player has picked from the replay.

**Comment by QQ2287991080** [comment]
Thank you for your reply。
Will there be specific spell in the future?

**Comment by fraxiinus** [comment]
No idea, you would have to ask Riot :p
